Erigon v3.2.0 fails to build on Arch Linux

Hello, I'm getting this error when installing from the AUR (that is quite fresh, thank you for maintening it so fresh)

```
==> Starting build()...
Building /home/mansuetus/AUR/erigon/src/erigon-3.2.0/build/bin/erigon
cd ./cmd/erigon && GOARCH=amd64 GOAMD64=v2  CGO_CFLAGS="-O2 -g  -DMDBX_FORCE_ASSERTIONS=0  -DMDBX_DISABLE_VALIDATION=0  -DMDBX_ENV_CHECKPID=0  -D__BLST_PORTABLE__ -Wno-unknown-warning-option -Wno-enum-int-mismatch -Wno-strict-prototypes -Wno-unused-but-set-variable -O3" CGO_LDFLAGS="-Wl,-O1 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,-z,pack-relative-relocs -flto=auto -O3 -g" GOPRIVATE="github.com/erigontech/silkworm-go" go  build -trimpath -buildvcs=false -ldflags "-X github.com/erigontech/erigon/db/version.GitCommit=480879a70da45bc0718fbc243f0fb37be79e4d49 -X github.com/erigontech/erigon/db/version.GitBranch=master -X github.com/erigontech/erigon/db/version.GitTag=v3.2.0"  -tags , -o /home/mansuetus/AUR/erigon/src/erigon-3.2.0/build/bin/erigon
go: downloading github.com/felixge/fgprof v0.9.5
go: downloading github.com/urfave/cli/v2 v2.27.7
go: downloading github.com/spf13/cobra v1.10.1
go: downloading golang.org/x/sys v0.36.0
go: downloading github.com/spf13/pflag v1.0.10
go: downloading golang.org/x/sync v0.17.0
go: downloading github.com/prometheus/client_golang v1.23.2
go: downloading github.com/prometheus/client_model v0.6.2
go: downloading golang.org/x/exp v0.0.0-20250911091902-df9299821621
go: downloading golang.org/x/time v0.13.0
go: downloading github.com/erigontech/mdbx-go v0.39.9
go: downloading google.golang.org/grpc v1.75.1
go: downloading google.golang.org/protobuf v1.36.9
go: downloading golang.org/x/net v0.44.0
go: downloading github.com/RoaringBitmap/roaring/v2 v2.10.0
go: downloading golang.org/x/crypto v0.42.0
go: downloading github.com/containerd/cgroups/v3 v3.0.5
go: downloading go.uber.org/mock v0.6.0
go: downloading golang.org/x/text v0.29.0
go: downloading github.com/crate-crypto/go-eth-kzg v1.4.0
go: downloading github.com/stretchr/testify v1.11.1
go: downloading github.com/consensys/gnark-crypto v0.19.0
go: downloading github.com/ethereum/go-bigmodexpfix v0.0.0-20250911101455-f9e208c548ab
go: downloading github.com/cenkalti/backoff/v4 v4.3.0
go: downloading github.com/prometheus/common v0.66.1
go: downloading github.com/prometheus/procfs v0.16.1
go: downloading go.opentelemetry.io/otel v1.37.0
go: downloading go.opentelemetry.io/otel/trace v1.37.0
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.7
go: downloading github.com/moby/sys/userns v0.1.0
go: downloading github.com/cilium/ebpf v0.16.0
go: downloading github.com/containerd/log v0.1.0
go: downloading github.com/go-chi/chi/v5 v5.2.3
go: downloading google.golang.org/genproto/googleapis/rpc v0.0.0-20250707201910-8d1bb00bc6a7
go: downloading github.com/99designs/gqlgen v0.17.78
go: downloading go.yaml.in/yaml/v2 v2.4.2
go: downloading github.com/bits-and-blooms/bitset v1.22.0
go: downloading go.opentelemetry.io/otel/metric v1.37.0
go: downloading github.com/dlclark/regexp2 v1.7.0
go: downloading github.com/vektah/gqlparser/v2 v2.5.30
go: downloading github.com/go-viper/mapstructure/v2 v2.4.0
Run "/home/mansuetus/AUR/erigon/src/erigon-3.2.0/build/bin/erigon" to launch erigon.
make: *** No rule to make target 'devnet'.  Stop.
==> ERROR: A failure occurred in build().
    Aborting...
all done!
```

--- a/src/aur.mjs
+++ b/src/aur.mjs
@@ -46,7 +46,7 @@ build() {
 
     export CGO_LDFLAGS="$LDFLAGS"
     export GIT_TAG="v${env("pkgver")}"
-    make erigon devnet downloader integration rpcdaemon sentry txpool sentinel caplin
+    make erigon downloader integration rpcdaemon sentry txpool sentinel caplin
 }
 
 package() {

